error.go

```diff
@@ -186,15 +186,6 @@ func IsTestReleaseTimeout(err error) bool {
 	return microerror.Cause(err) == testReleaseTimeoutError
 }
 
-var tillerInstallationFailedError = &microerror.Error{
-	Kind: "tillerInstallationFailedError",
-}
-
-// IsTillerInstallationFailed asserts tillerInstallationFailedError.
-func IsTillerInstallationFailed(err error) bool {
-	return microerror.Cause(err) == tillerInstallationFailedError
-}
```

Comment:
So this error isn't used anywhere but its the error we match on in cluster-operator. 

![picard-facepalm](https://user-images.githubusercontent.com/311527/61892524-4692b980-af0c-11e9-9fba-7e0851c8525e.jpg)